.vscode/tasks.json

```diff
@@ -79,6 +79,12 @@
       "type": "shell",
       "args": ["${input:circleCiId}"]
     },
+    {
+      "label": "Create Change Log",
+      "command": "./scripts/change-log-generator.js",
+      "type": "shell",
+      "args": ["-v", "${input:verboseOutput}", "-r", "${input:releaseOverride}"]
```

Comment:
Future Plan: Add a date parameter here that would let someone set the date through a task prompt. (Idea from Bryan :))